src/test-geogram-install2.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include <geogram/mesh/mesh.h>
+
+void toto_mesh()
+{
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this using-directive. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1003)